ui/src/views/user/delete-thankyou/DeleteThankyouView.vue

```diff
@@ -5,8 +5,14 @@
     tag="section"
   >
     <v-row justify="center">
-      <v-col cols="auto">
-        thank you view
+      <v-col cols="12">
+        <content-card 
+          title="Thank you for using our services" 
+          description="Your account has been successfully deleted. You can make a new account to use our services." 
+          :image="require('@assets/svg/man-woman-holding-mail.svg')"
+          :maxImageWidth="240"
```

Comment:
This is ok, but for next time, please remember that while prop is written in camelCase, it should be referred in kebab case when being used in templates XD so `max-image-width` instead of maxImageWidth. This has to do with html not being case sensitive sometime.

Reply:
fixed :<